css/index.css

```diff
@@ -1 +1,856 @@
+		width: 93%;
+		border-top: 3px solid black;
+	}
+
+	.bottom-left {
+		width: 47%;
+	}
+
+	.bottom-left h3 {
+		line-height: 2;
+	}
+
+	.bottom-left ul {
+		margin-top:20px ;
+		line-height: 2;
+		list-style-type: square;
+		padding-left: 20px;
+	
+	}
+
+	.services-info-img{
+ 		width: 48%;
+	}
+
+.copyright { 
+	text-align: center;
+	margin-top: 30px;
+}
+
+/* About page css */
+.header-about-page {
+    height: 360px;
+}
+
+header .container-about-page img {
+    width: 100%;
+    margin: 20px 0;
+}
+
+header .container-about-page p {
+    margin-bottom: 20px;
+}
+
+header .container-about-page {
+    border-bottom: 1px dotted black;
+}
+
+header .container-about-page h1 {
+    font-size: 1.8rem;
+}
+
+header .container-about-page p {
+    font-size: 1.6rem;
+}
+
+.company-info {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+.company-info h1 {
+    font-size: 1.8rem;
+}
+
+.company-info p{
+    font-size: 1.6rem;
+}
+
+.company-info a{
+    font-size: 1.6rem;
+}
+
+.company-info div{
+    box-sizing: content-box;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    height: 500px;
+    width: 48%;
+    padding: 30px 0;
+    border-bottom: 1px dotted black
+}
+
+.company-info div img{
+    width:100%;
+}
+
+.company-info a{
+    width: 37%;
+    padding: 7px 0;
+    text-align: center;
+    border-radius: 12px;
+    border: 2px solid black;
+    text-decoration-line: none;
+    color: black;
+    
+}
+
+.company-info .bottom-div {
+    box-sizing: content-box;
+    padding-top: 50px;
+}
+
+.work-together{
+    padding: 30px 0;
+    font-size: 1.6rem;
+}
+
+/* contact page below */
+
+.contact-nav {
+    padding-top: 20px;
+}
+
+.above-header-image {
+    display: block;
+    width: 100%;
+    height: 250px;
+    object-fit: cover;
+    margin: 20px 0;
+}
+
+.contact-header{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 25px 0px;
+}
+
+.contact-header .header-left-text{
+    display:flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 589px;
+    width: 45%;
+    order:1;
+}
+
+.contact-header .header-left-text .words {
+    border: 1px solid black;
+    padding: 10px 10px;
+}
+
+.contact-header .header-left-text div img {
+    border: 1px solid black;
+    width: 100%;
+    order: 1;
+}
+
+.contact-header .contact-form {
+    width:50%;
+    text-align: center;
+    padding: 10px 10px;
+    border: 1px solid black;
+}
+
+.contact-header .contact-form .query-input {
+    width: 80%;
+    height: 200px;
+}
+
+
+
+
+@media (max-width: 800px){
+.container {
+    
+    width: 95%;
+}
+
+body nav {
+    flex-wrap: wrap-reverse;
+    height:auto;
+    justify-content: space-around;
+}
+
+nav a{
+    width: 15%;
+    text-align: center;
+    align-items: flex-end;
+    padding: 10px 0;
+}
+
+nav img {
+    margin: 30px 0;
+}
+
+header {
+    flex-direction: column;
+    height:417px;
```

Comment:
Don't forget a space after the `:`, its a good convention to follow, as it makes everything easier to read.